relationship_server.py

```diff
@@ -13,38 +13,45 @@
 logger = logging.getLogger(__name__)
 
 
-'''
-    Utliity method to convert keys and values in a dictionary to lowercase.
-'''
-def makeDictLowercase(d):
-    lowerCaseDict = dict()
+def make_dict_lowercase(d):
+    """
+        Utliity method to convert keys and values in a dictionary to lowercase.
+    """
+    lower_case_dict = dict()
     for k in d.keys():
-        lowerCaseDict[k.lower()] = d[k].lower()
-    return lowerCaseDict
-
-'''
-    Utility to convert string to Title Case
-    Input: this is a sentence.
-    Output: This Is A Sentence.
-'''
-def toTitleCase(string):
+        lower_case_dict[k.lower()] = d[k].lower()
+    return lower_case_dict
+
+
+def to_title_case(string):
+    """
+        Utility to convert string to Title Case
+        Input: this is a sentence.
+        Output: This Is A Sentence.
+    """
     return " ".join(w.capitalize() for w in string.split(" "))
```

Comment:
What can we do to use `.title()` from inbuilt Python string operations? 
e.g. 
```python
'There     is a way'.title()
```

Reply:
I was trying to avoid a scenario where `This is the the author's view` would become `This Is The Author'S View`. But I guess we will not encounter such a scenario and it is better to use `.title()`. Have made the changes to incorporate the same